app/components/FilterableTable/FilterableTablePagination.tsx

```diff
@@ -87,19 +72,18 @@ const FilterableTablePaginationComponent: React.FunctionComponent<Props> = (
       <div className="pagination">
         <Dropdown>
           <Dropdown.Toggle variant="outline-dark" id="dropdown-basic">
-            Results Per Page: <strong>{maxResultsPerPage}</strong>
+            Items Per Page: <strong>{pageSize}</strong>
           </Dropdown.Toggle>
 
           <Dropdown.Menu>
-            <Dropdown.Item onClick={() => handleMaxResultsChange(20)}>
-              20
-            </Dropdown.Item>
-            <Dropdown.Item onClick={() => handleMaxResultsChange(50)}>
-              50
-            </Dropdown.Item>
-            <Dropdown.Item onClick={() => handleMaxResultsChange(100)}>
-              100
-            </Dropdown.Item>
+            {PAGE_SIZE_OPTIONS.map((option) => (
+              <Dropdown.Item
+                ref={option}
+                onClick={() => onPageSizeChange(option)}
+              >
+                {option}
+              </Dropdown.Item>
+            ))}
```

Comment:
Since this is static we should compute it outside of the rendering loop

Reply:
`onPageSizeChange` isn't static though

Reply:
oh right!